.github/workflows/rust.yml

```diff
@@ -141,7 +141,7 @@ jobs:
       - uses: actions/upload-artifact@v1
```

Comment:
I saw there is also a version 2 
`actions/upload-artifact@v2`

Reply:
Ok let me update this then